fenix/app/metrics.yaml

```
@@ -1271,7 +1271,7 @@ onboarding:
   turn_on_notifications:
     type: event
     description: |
-      User tapped on turn on notifications button in juno onboarding.
+      User tapped on turn on notifications button in  onboarding.
```

```suggestion
      User tapped on turn on notifications button in onboarding.
```
This happens a few times here.

---

Good catch, fixed 👍